package exceptions;
import  exceptions.*;
public class ejercici6 {
    public static void main(String[] args){
    //throw new prova_exceptions("hola"); sin el try catch el compilador no compila el programa
    }
}
